docs/sampling.asciidoc

```diff
@@ -45,6 +45,14 @@ Again, the sample rates of `Service B` and `Service C` are ignored.
 
 image::./images/dt-sampling-example-2.png[Distributed tracing and head based sampling example two]
 
+**OpenTelemetry with head-based sampling**
+
+Head-based sampling is implemented in the APM agents and SDKs, and
+requires the sample rate to be propagated between services and the APM Server.
+This functionality is not currently supported by OpenTelemetry,
+which results in inaccurate APM throughput, latency, and error metrics.
+OpenTelemetry users should use tail-based sampling instead.
```

Comment:
I don't know about this language - TBS isn't always feasible. Maybe replace "should use" with something like "may use", or "should consider using"

Reply:
👍 Thanks, will do.

Reply:
```suggestion
OpenTelemetry users should consider using tail-based sampling instead.
```